src/node_buffer.cc

```diff
@@ -701,6 +928,50 @@ void IndexOfNumber(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void IsNull(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (!HasInstance(args[0]))
+    return env->ThrowTypeError("first arg should be a Buffer");
+
+  ARGS_THIS(args[0].As<Object>())
```

Comment:
@domenic FYI, completed the audit of `src/node_buffer.cc`. Most all functions are hidden behind `process.binding()`. Which we give no guarantees about. Only one publicly exposed is `Copy()`. Which does have an `HasInstance()` check.

Though guess for a full audit we'll need to review every function call that accepts a buffer.